.claude/skills/bt-review/SKILL.md

```
@@ -16,34 +16,37 @@ Review current changes against project rules and quality standards.
 ## Steps
 
 1. **Gather changes**
-   - Run `git diff` to see all unstaged modifications
-   - Run `git diff --cached` to see staged changes
-   - List which files were modified and what changed
+
+- Run `git diff` to see all unstaged modifications
+- Run `git diff --cached` to see staged changes
+- List which files were modified and what changed
 
 2. **Run automated checks**
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)